module/VuFind/src/VuFind/Db/Entity/Feedback.php

```diff
@@ -40,7 +39,12 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="feedback", indexes={@ORM\Index(name="user_id", columns={"user_id"}), @ORM\Index(name="created", columns={"created"}), @ORM\Index(name="status", columns={"status"}), @ORM\Index(name="form_name", columns={"form_name"})})
+ * @ORM\Table(name="feedback",
+ * indexes={@ORM\Index(name="user_id", columns={"user_id"}),
```

Comment:
We might not need this user_id index because it's also a foreign key (I think I may have manually added this one myself at some point, but that may have been a mistake).